Automattic/pocket-casts-ios
# Apple SSO: Error Display

| 📘 Project: #381 | 
|:---:|

Adds an error label to the initial login screen to inform the user that an error has occurred and allows them to choose their recovery path. 

## To test

1. Enable the Feature Flag `signInWithApple`
2. Set your scheme to Debug that will force an error when trying to use the API
3. Login with SSO
4. Expect to see an Error label above the login options. 
5. Select "Log In"
6. Go back
7. Expect the error label to have been cleaned up. 

<img width="300" src="https://user-images.githubusercontent.com/3384451/196468363-07f8d01a-5451-443c-af38-6832037b4cdb.png">


## Checklist
